server/routes/dashboard.ts

```diff
@@ -159,14 +171,28 @@ export default function dashboard(service: Services): Router {
       searchLocations = location
     }
 
+    let searchStatuses: Status | Status[]
+    // Replace status mappings if RO viewing page
+    if (userRoles.includes(roleReadWrite) && !userRoles.includes(roleApproveReject)) {
+      if (typeof incidentStatuses === 'object') {
```

Comment:
`Array.isArray(incidentStatuses)` is closer cos you use this var as an array on the line below